.github/workflows/reusable-build.yml

```diff
@@ -38,6 +38,11 @@ jobs:
           - common
           - extra
           - nvidia
+        include:
+          - kernel_flavor: coreos-stable
+            cfile_suffix: zfs
```

Comment:
you also need to include version. Includes need all parts declared for some reason.

Reply:
yeah, I went back to excludes... it feels cleaner and is certainly more consistent with the existing approach.